src/dialog.tsx

```diff
@@ -191,22 +200,26 @@ const Dialog: React.FC<DialogProps> = (props) => {
             </div>
 
             <Box paddingTop={isTabletOrSmaller ? 24 : 32}>
-                <ButtonLayout>
-                    {destructive ? (
-                        <ButtonDanger tabIndex={1} {...mainButtonProps} /> // eslint-disable-line jsx-a11y/tabindex-no-positive
-                    ) : (
-                        <ButtonPrimary tabIndex={1} {...mainButtonProps} /> // eslint-disable-line jsx-a11y/tabindex-no-positive
-                    )}
-                    {showCancel && !!handleCancel && (
-                        <ButtonSecondary
-                            tabIndex={2} // eslint-disable-line jsx-a11y/tabindex-no-positive
-                            onPress={handleCancel}
-                            dataAttributes={{testid: 'dialog-cancel-button'}}
-                        >
-                            {cancelText}
-                        </ButtonSecondary>
-                    )}
-                </ButtonLayout>
+                <ButtonGroup
```

Comment:
There's one missing thing in this PR, and is the need to support a link (up to 3 actions in dialog), what will be the best way to support this? adding all the needed props of link (to, href, newTab....) ? add a prop that receive a ButtonLink component? this dialog component works only in web side (no native dialog) so.,.. do we really need to support this new link?